.github/ISSUE_TEMPLATE/page-report-ru.yml

```
@@ -5,58 +5,58 @@ body:
   - type: markdown
     attributes:
       value: |
-        ### Before you start
+        ### Прежде чем начать
 
-        **Want to change this page yourself?** This content is open source!
-        ↩ Go back and use the _Edit on GitHub_ link on the page.
+        **Готовы внести правки самостоятельно?** Справочник MDN может редактировать каждый!
+        ↩ Для этого вернитесь обратно в статью и кликните на ссылку _Edit on GitHub_.
 
-        **Is your issue about the browser compatibility table?**
-        ↩ Go back and use the _Report problems with this compatibility data on GitHub_ link on the page.
+        **В таблице с браузерной совместимостью ошибка?**
+        ↩ Такие ошибки правит отдельная команда. Пожалуйста, вернитесь обратно в статью и кликните на ссылку _Report problems with this compatibility data on GitHub_, чтобы сообщить им.
 
-        **Need help with a browser?**
-        🙋 To get help with [Firefox](https://support.mozilla.org/en-US/kb/file-bug-report-or-feature-request-mozilla), [Chrome](https://support.google.com/chrome/answer/95315?hl=en-GB&ref_topic=7439544), [Safari](https://www.apple.com/feedback/safari.html), or another browser, check the browser's support site.
+        **Технология из статьи не работает в вашем браузере?**
+        🙋 Багтрекеры основных браузеров: [Firefox](https://support.mozilla.org/en-US/kb/file-bug-report-or-feature-request-mozilla), [Chrome](https://support.google.com/chrome/answer/95315?hl=en-GB&ref_topic=7439544), [Safari](https://www.apple.com/feedback/safari.html). Если вы используете другой браузер, пожалуйста, найдите его багтрекер самостоятельно.
 
         ---
   - type: input
     id: mdn-url
     attributes:
       label: MDN URL
-      description: Set automatically. Do not modify.
+      description: Заполнится само, пожалуйста, не правьте.
     validations:
       required: true
   - type: input
     id: section
     attributes:
-      label: What specific section or headline is this issue about?
+      label: О какой части статьи или абзаце идёт речь?
   - type: textarea
     id: problem
     attributes:
-      label: What information was incorrect, unhelpful, or incomplete?
+      label: Что не так?
```

Мне кажется, что нужно оставить как в оригинале, чтобы лучше донести информацию о том, что в первую очередь для нас ценно:
```suggestion
      label: Что было написано некорректно, неполно или непонятно?
```

---

Согласен, но "было" не смотрится, ведь текст всё ещё остаётся в статье. Сделал подход (см. ниже)